README.md

```diff
@@ -80,7 +80,7 @@ repositories {
 
 
 dependencies {
-    compile 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.1.0'
+    compile 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.4'
```

Comment:
[This one](https://github.com/hannesa2/paho.mqtt.android/commit/718c97d57a8cef98dd4191074c42ae1500ccbba8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44), has the most un-merged pull requests included